### ISSUE TITLE
add authentiication

### ISSUE THREAD

**Issue by heralight** [mean-expert-official/fireloop-todo-example#6]
Hi,

I like your fireloop.io idea, and in your tutorial and wiki you talk about authentication with passport and loopback.
I fail to reproduce that, perhpas do you some implementation accessible?

Thank you,

**Comment by jonathan-casarrubias** [comment]
Hi @heralight thanks for reaching out,

Hey so, first of all I'm glad you find the fireloop api attractive, second... The sdk builder works in both scenarios, just LoopBack REST API and FireLoop.

The examples I have in the wiki should work for both, since both use the same authentication mechanism, once you get the token and use the authService to add it, then FireLoop should be able to use that token and create a web socket connection and implement acl security rules etc.

So, there is just one thing, you should authenticate and set the token prior you create an instance of fireloop.

In other words, by the moment you create an instance of fireloop, you should have already have a token set in your project.

I do not have a working example for that, but I will definitely be adding one as soon as I release the FireLoop CLI  1 into beta and the SDK Builder  2.1 into release candidate, which should happen in the next following days.

Cheers
Jon

**Comment by tguelcan** [comment]
> I do not have a working example for that, but I will definitely be adding one as soon as I release the FireLoop CLI 1 into beta and the SDK Builder 2.1 into release candidate, which should happen in the next following days.

Hey @jonathan-casarrubias, do you have a Example?

**Comment by ptesar-xyndata** [comment]
Hello @jonathan-casarrubias we need example :( THANK YOU VERY MUCH

**Comment by certifirm** [comment]
@jonathan-casarrubias  One example will be amazing.  Thanks in advance.

**Comment by jonathan-casarrubias** [comment]
Hey guys I just released a documentation for authentication

https://github.com/mean-expert-official/fireloop.io/wiki/Authentication

The security part is really complex and wide, I just talked about basics I will extend as soon as I can. The passport implementation would combine with this other documentation https://github.com/mean-expert-official/loopback-sdk-builder/wiki/7.-Social-Passport-Integration

**Comment by ptesar-xyndata** [comment]
Excelent Jonathan, thank you! I have started to play also with the new usage of ngrx from Brannon! I will let you know when it works!
